app/controllers/api/v1/projects_controller.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class Api::V1::ProjectsController < ApplicationController
+  include JWTHelper
+
+  skip_before_action :verify_authenticity_token
+  before_action :authenticate_jwt
+
+  # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
+  def create
+    populate_project_from_forge
+    @project.enlistments_attributes = { '0' => { 'code_location_attributes' => { 'scm_type' => 'git',
+                                                                                 'url' => params[:url],
+                                                                                 'branch' => params[:branch] } } }
+    @project.editor_account = current_user
+    @project.assign_editor_account_to_associations
+    @project.manages.new(account: current_user) if params[:managed_by_creator].to_bool
+    @project.save!
+    create_code_location_subscription if @project.enlistments.exists?
+  rescue StandardError => e
+    render json: "Bad params #{e}", status: :bad_request
+  end
+  # rubocop:enable Metrics/AbcSize, Metrics/MethodLength
```

Comment:
Move lines 12-18 to a new method called *save_project*. That should take care of the rubocop warnings. 
We should try to avoid disabling rubocop wherever possible.

Reply:
The **ProjectBuilder** class can be used here instead. The initializer in ProjectBuilder must return `@project` and you can then use:
```
@project = ProjectBuilder.new(account, hsh)
```
Will need some testing to see if it fits. Looks like it would work.